### PR TITLE
small cosmetic tweaks

### PR DESCRIPTION
**Pull request by AntonioMeireles**
just cosmetics. no funtional changes.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -9,18 +9,18 @@ Vagrant.require_version ">= 1.5.0"
...
-NUM_INSTANCES = Integer(cluster['num_nodes'])
-
 BASE_IP_ADDR = ENV['BASE_IP_ADDR'] || String(cluster['ip_addr_prefix'])
 ETCD_DISCOVERY = "#{BASE_IP_ADDR}.101"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = String(cluster['box'])
+  box = String(cluster['box'])
+  config.vm.box = "#{box}"
+  config.vm.box_url = "https://vagrantcloud.com/%s" % "#{box}"
```
Comment (YungSang) [inline Vagrantfile]: We don't need `box_url` as long as we use my boxes in the atlas (formerly vagrant cloud).

Comment (AntonioMeireles) [inline Vagrantfile]: i know... my point was to make this minimally pluggable, portable, extendable i.e. if someone has made some other boxes for some reason (to target another hypervisor, whatever) most hooks would be already in place. anyway will comment that line. it's really a no-op now :smile:

Comment (YungSang) [inline Vagrantfile]: I have no intent to use other boxes with this configuration. I will not guarantee those use.
If someone wants to do something, one can fork and modify it.

Comment (YungSang) [inline Vagrantfile]: I want to keep it simple here for general purpose.

File: Vagrantfile [hunk 2]
```diff
@@ -9,18 +9,18 @@ Vagrant.require_version ">= 1.5.0"
 # Read YAML file with cluster details
 cluster = YAML.load_file('cluster.yaml')
 
-NUM_INSTANCES = Integer(cluster['num_nodes'])
```
Comment (YungSang) [inline Vagrantfile:12]: I do like to use `NUM_INSTANCES` for code visibility.
And the box as well like `COREOS_VAGRNAT_BOX`.

**Pull request merged**